packages/palette/src/svgs/BellFillIcon.ios.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react"
+import { Path } from "react-native-svg"
+import { color } from "../helpers"
+import { Icon, IconProps } from "./Icon"
+
+/** BellFillIcon */
+export const BellFillIcon: React.SFC<IconProps> = props => {
```

Comment:
@lilyfromseattle it looks like since https://github.com/artsy/palette/pull/605 we no longer need separate files for the iOS and web versions of these icons. Can you verify that it works without this?

Reply:
Ahhh I didn't realize. Yup I'll check now.

Reply:
Looks good! just deleted them and pushed